charter.html

```diff
@@ -0,0 +1,333 @@
+      If there is sustained opposition without changes to the issue, and the
+      proposer still wishes to proceed, the Chairs will hold a vote in the
+      Community Group. Each organization will have one vote. This is only to
+      be used as a last resort if consensus can’t be reached and no progress
+      is being made.
+    </p>
+    <p>
+      Any decisions reached at any meeting are tentative and should be recorded
+      in a GitHub Issue. Any group participant may object to a decision reached
+      at an online or in-person meeting within 7 days of publication of the
+      decision provided that they include clear technical reasons for their
+      objection. The Chairs will facilitate discussion to try to resolve the
+      objection according to the <a href="#decision">decision process</a>.
+    </p>
+    <p>
+      It is the Chairs' responsibility to ensure that the decision process is
+      fair, respects the consensus of the CG, and does not unreasonably favour
+      or discriminate against any group participant or their employer.
+    </p>
+    <h2 id="chairs">
+      Community Group Chairs
+    </h2>
+    <h3 id="out-of-scope">
+      Chair Responsibilities
+    </h3>
+    <p>
+      The Chairs of the Community Group have the following responsibilities. In
+      the case of multiple Chairs, they should ensure that at least one Chair
+      is responsible for each item.
+      <ul>
+        <li>Organize/run virtual and face-to-face meetings/conferences.</li>
+        <li>Encourage/solicit participation in the Community Group.</li>
+        <li>Maintain GitHub organizations/repositories.</li>
+        <li>Triage and respond to general issues on the repositories.</li>
+        <li>Ensure compliance with the W3C CG/BG Process and CLA.</li>
+        <li>Uphold the processes and scope determined by the CG charter.</li>
+        <li>Mediate contentious CG issues.</li>
+        <li>Determine whether the CG has consensus on actions/issues.</li>
+        <li>Coordinate with other W3C and external groups.</li>
+      </ul>
+    </p>
+    <h3 id="chair-selection">
+      Chair Selection
+    </h3>
+    <p>
+      The Community Group can have up to three Chairs, no more than one per
+      organization. Additional Chairs may be appointed by unanimous consent of
+      the then-current Chairs.
+    </p>
+    <p>
+      If the number of Chairs falls to zero, or if four Community Group
+      Participants — from different organizations — call for an election, the
+      group must use the following process to elect a new Chair, as follows.
+    </p>
+    <p>
+      <b>Participants announce their candidacies</b>: Participants have 14 days
+      to announce their candidacies. No two candidates can be from the same
+      organization. If there are three or fewer candidates, they become the
+      Chairs. If there are more than three candidates, a vote is held. (If
+      there are no candidates, the Community Development Lead should be
```

Comment:
Can we link the term "Community Development Lead" to more information on this role and how to contact them?